package.json

```diff
@@ -38,9 +38,8 @@
     "file-loader": "^0.9.0",
     "highlight.js": "^9.12.0",
     "identity-obj-proxy": "^3.0.0",
-    "jest-cli": "^20.0.0",
+    "jest-cli": "^23.6.0",
     "jest-prop-type-error": "^1.1.0",
```

Comment:
all packages listed in this file are not required for packaging, see https://github.com/theforeman/foreman-packaging/blob/rpm/develop/update-requirements#L8